src/components/LeftDrawer/LeftDrawer.tsx

```diff
@@ -0,0 +1,182 @@
+            }`}
+            data-testid="orgsBtn"
+            onClick={(): void => {
+              history.push('/orglist');
+            }}
+          >
+            <div className={styles.iconWrapper}>
+              <OrganizationsIcon
+                stroke={`${
+                  screenName === 'Organizations'
+                    ? 'var(--bs-white)'
+                    : 'var(--bs-secondary)'
+                }`}
+              />
+            </div>
+            {t('organizations')}
+          </Button>
+          {userType === 'SUPERADMIN' && (
+            <Button
+              variant={screenName === 'Requests' ? 'success' : 'light'}
+              className={`${
+                screenName === 'Requests' ? 'text-white' : 'text-secondary'
+              }`}
+              onClick={(): void => {
+                history.push('/requests');
+              }}
+              data-testid="requestsBtn"
+            >
+              <div className={styles.iconWrapper}>
+                <RequestsIcon
+                  fill={`${
+                    screenName === 'Requests'
+                      ? 'var(--bs-white)'
+                      : 'var(--bs-secondary)'
+                  }`}
+                />
+              </div>
+              {t('requests')}
+            </Button>
+          )}
+          {userType === 'SUPERADMIN' && (
+            <Button
+              variant={screenName === 'Roles' ? 'success' : 'light'}
+              className={`${
+                screenName === 'Roles' ? 'text-white' : 'text-secondary'
+              }`}
+              onClick={(): void => {
+                history.push('/roles');
+              }}
+              data-testid="rolesBtn"
+            >
+              <div className={styles.iconWrapper}>
+                <RolesIcon
+                  fill={`${
+                    screenName === 'Roles'
+                      ? 'var(--bs-white)'
+                      : 'var(--bs-secondary)'
+                  }`}
+                />
+              </div>
+              {t('roles')}
+            </Button>
+          )}
+        </div>
+        <div style={{ marginTop: 'auto' }}>
+          {data === undefined ? (
+            <div
+              className={`${styles.profileContainer} shimmer`}
+              data-testid="loadingProfile"
+            />
+          ) : (
+            <button
+              className={styles.profileContainer}
+              data-testid="profileBtn"
+              onClick={(): void => {
+                toast.success('Profile page coming soon!');
+              }}
+            >
+              <div className={styles.imageContainer}>
+                {data && data?.user?.image ? (
+                  <img src={data?.user?.image} alt={`profile picture`} />
+                ) : (
+                  <img
+                    src={`https://api.dicebear.com/5.x/initials/svg?seed=${data?.user?.firstName} ${data?.user?.lastName}`}
```

Comment:
Should there be a space between the firstname and lastname in the url?

Reply:
Yes @ttaylor92 , there should be some space, or we can put `%20` between the names, otherwise the initials will be shown of first two characters of users first name.

Example-
- **With space [https://api.dicebear.com/5.x/initials/svg?seed=Rishav%20Jha]**
![https://api.dicebear.com/5.x/initials/svg?seed=Rishav%20Jha](https://api.dicebear.com/5.x/initials/svg?seed=Rishav%20Jha)

- **Without space [https://api.dicebear.com/5.x/initials/svg?seed=RishavJha]**
![https://api.dicebear.com/5.x/initials/svg?seed=RishavJha](https://api.dicebear.com/5.x/initials/svg?seed=RishavJha)

Should I add `%20` in the code also ?